modules/core/shared/src/main/scala/lucuma/core/model/SpatialProfile.scala

```diff
@@ -8,18 +8,19 @@ import cats.syntax.all._
 import lucuma.core.math.Angle
 import lucuma.core.optics.SplitMono
 
+// TODO: Can we safely remove this file?
```

Comment:
Removing it will require changes downstream, so I'd say yes, let's remove it to for the changes